how-to-guides/hello-neptune/notebooks/hello_neptune.ipynb

```diff
@@ -274,8 +308,105 @@
    "source": [
     "## Explore the run in the Neptune app\n",
     "\n",
-    "* The graphs of `random_training_metric` and `other_random_training_metric` are displayed in the **Charts** dashboard on the left\n",
-    "* Neptune also logs your system hardware consumption. These are visible under the **Monitoring** dashboard"
+    "* Uploaded source files are present in the **Source code** dashboard\n",
+    "* The graphs of `accuracy` and `loss` are displayed in the **Charts** dashboard\n",
+    "* Neptune also logs your system hardware consumption. These are visible under the **Monitoring** dashboard\n",
```

Comment:
Rephrasing this a bit, since the functionality is not hidden like before

```suggestion
    "* You can view your system hardware consumption in the **Monitoring** dashboard\n",
```